src/server/rest/v1/service/UserService.ts

```diff
@@ -83,6 +84,10 @@ export default class UserService {
     const errorCodes: Array<StartTransactionErrorCode> = [];
     // Check Billing errors
     await UserService.checkBillingErrorCodes(action, req.tenant, req.user, user, errorCodes);
+    // Add error if EULA are not accepted (use case -> at user import)
+    if (!user.eulaAcceptedOn) {
```

Comment:
Is it possible that the user request the default car/tag... without logging in and accepting the EULA?

Reply:
deleted ✅